system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/participant/EndToEndTransferParticipant.java

```diff
@@ -233,6 +233,21 @@ public Map<String, String> controlPlanePostgresConfiguration() {
         return baseConfiguration;
     }
 
+    public Map<String, String> controlPlanePostgresDefaultConfiguration() {
```

Comment:
wouldn't this make the `controlPlanePostgresConfiguration` in line 204ff obsolete?

Reply:
same as I was saying, we could keep just a single postgres e2e transfer test class with the default datasource.

Reply:
Done